### PR TITLE
Rewrite with Typescript

### PR DESCRIPTION
**Pull request by adeyahya**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,8 +45,5 @@
   "dependencies": {
     "@babel/runtime": "^7.12.13",
```
Comment (ri7nz) [inline package.json:53]: why `@babel/runtime` in `dependencies` ?

File: src/index.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// string
+export * from "./string/words";
+export * from "./string/camelCase";
+export * from "./string/snakeCase";
+
+// object
+export * from "./object/keysCamelCase";
```
Comment (ri7nz) [inline src/index.ts:7]: why not to make entry for `object` (`string` too)
```js
// src/object/index.ts
export * "./keysCamelCase";
```
and 
```js
// src/index.ts
export * from "./object"
```

**Review by ri7nz — COMMENTED** [review]
its `possible` to make `docs` in `github-action` ? @adeyahya

**Comment by adeyahya** [comment]
> its `possible` to make `docs` in `github-action` ? @adeyahya

I already setup docs on github pages, https://warungpintar.github.io/ninshu/

File: src/index.ts (hunk 2 above)
Comment (adeyahya) [reply src/index.ts:7]: yep Im considering it too

File: package.json (hunk 1 above)
Comment (adeyahya) [reply package.json:53]: supporting new feature of js in the old environment without polluting global object, I think

Comment (ri7nz) [reply package.json:53]: I think if we need `core-js` as a helper, but if have break we can go with it 
image

Comment (adeyahya) [reply package.json:53]: idk if we need core-js right now, it has a lot of polyfills, I think for now it's ok to only use `@babel/runtime` and `@babel/plugin-transform-runtime`

**Pull request merged**